libs/VideoKit/tests/VideoKit_test.cpp

```diff
@@ -51,6 +51,14 @@ TEST_F(VideoKitTest, displayImage)
 	video_kit.displayImage(temp_file_path);
 }
 
+TEST_F(VideoKitTest, displayImageWithBackground)
+{
+	EXPECT_CALL(mock_event_flags, set(VideoKit::flags::STOP_VIDEO_FLAG)).Times(2);
+	EXPECT_CALL(mock_corevideo, displayImage).Times(2);
```

Comment:
Perfect, eventually enforce the call of `displayImage` only once for the test above